README.template.md

```diff
@@ -1,5 +1,14 @@
 {{ ossHeader }}
 
+## Preview Features
+
+This SDK contains APIs for interacting with collection data structures: Lists, Sets, and Dictionaries.  These APIs
+are currently in preview.  If you would like to request early access to the data structure APIs, please contact us
+at `support@momentohq.com`.
```

Comment:
Should we tell them what information to send us?

Reply:
It's just their e-mail address.  And hopefully this will go away soon :)